### ISSUE TITLE
UID, not label of new entities not shown in Permission Manager

### ISSUE THREAD

**Issue by mobilegenome** [molgenis/molgenis#8737]
### How to Reproduce

Create a new entity in the Metadata Manager.
 
version 8.1.8 built on 2019-06-20 19:50 

### Expected behavior

The label of the new entity should be shown in the "Entity Class Permission" tab of the Permission manager. 

### Observed behavior

Only the UID of the newly created entity is shown. Entities created via EMX import are shown correctly.

**Comment by fdlk** [comment]
This is tricky, the label is not unique and you want to be sure what entity you're handing out permissions on. The ID for entity types created in the manager is not user friendly.

**Comment by mobilegenome** [comment]
Agreed that you want to be sure what entity is being handled in the permission manager. In my opinion showing the label and maybe a warning if the label is duplicate in the database could be a solution.

**Comment by mswertz** [comment]
I would prefer that we start scope entity labels to be unique within groups, and that if there are multiple we also show groups label to disambiguate. In other words: groups behave as data schemas (or namespace if you like that term better)
